src/screens/signUp/validator/constant.js

```diff
@@ -0,0 +1,30 @@
+export const validationRules = [
```

Comment:
sir these validations are only rules to make sure the entered values pass the regex test, if the entered value is empty it will still generate an error. 
![image](https://github.com/FullStackkDev/Bookwriter-frontend/assets/36064269/58ba3d49-354e-4fc2-a4b2-c8700feb3ab1)